pkgs/by-name/me/memcachedTestHook/memcached-test-hook.sh

```diff
@@ -0,0 +1,27 @@
+preCheckHooks+=('memcachedStart')
+postCheckHooks+=('memcachedStop')
+
+
+memcachedStart() {
+  if [[ "${memcachedTestPort:-}" == "" ]]; then
+    memcachedTestPort=11211
+  fi
+
+  echo 'starting memcached'
+
+  # Note about Darwin: unless the output is redirected, the parent process becomes launchd instead of bash.
+  # This would leave the memcached process running in case of a test failure (the postCheckHook would not be executed),
+  # hanging the Nix build forever.
+  memcached -p "$memcachedTestPort" >/dev/null 2>&1 &
+  MEMCACHED_PID=$!
+
+  while ! (echo 'quit' | nc localhost "$memcachedTestPort") ; do
+    echo 'waiting for memcached to be ready'
```

Comment:
I think we can just print that once, otherwise it could become very spammy on busy builders.

Reply:
Fixed, as well as in redisHook